ca_cdk_constructs/storage/aurora_fast_clone.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any
+from aws_cdk import CfnOutput, Token, Stack
+from constructs import Construct
+from aws_cdk.aws_ec2 import SubnetSelection, IVpc, SecurityGroup, IConnectable, Port
+from aws_cdk.aws_rds import (
+    CfnDBCluster,
+    DatabaseCluster,
+    SubnetGroup,
+    CfnDBParameterGroup,
+    CfnDBClusterParameterGroup,
+    CfnDBInstance,
+)
+
+
+class AuroraFastClone(Construct):
+    """Creates Aurora fast clone"""
```

Comment:
As well as creating the fast clone, also need an instance method to do in-place version upgrade, where you supply the target engine version and (optionally - only if changing) target `instance_params` and `cluster_params`?

Reply:
Not really thought how best to do this though - might be better just to access the underlying cluster/instance resources directly and modify them. Because I'm guessing you'd have to detach the clone so it was stand-alone before you upgrade it, so it's not really a clone any more...

Reply:
But we do need to think about how we'll do this, given the upcoming 10 -> 11 upgrade.

Reply:
yes, agree, I'll add engine version parameter